AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/InsertAdPlayerActivity.java

```diff
@@ -61,10 +67,15 @@ public void onCreate(Bundle savedInstanceState) {
 
     //Initialize the player
     OoyalaPlayerLayout playerLayout = (OoyalaPlayerLayout) findViewById(R.id.ooyalaPlayer);
-    player = new OoyalaPlayer(PCODE, new PlayerDomain(DOMAIN));
+    Options opts = new Options.Builder().setUseExoPlayer(true).build();
```

Comment:
exoplayer isn't specifically necessary for this.  I'd be okay if you made a new opts that is default, and commented this out